dll/win32/browseui/shellbrowser.cpp

```diff
@@ -1726,6 +1728,18 @@ void CShellBrowser::UpdateViewMenu(HMENU theMenu)
         SetMenuItemInfo(theMenu, IDM_VIEW_TOOLBARS, FALSE, &menuItemInfo);
     }
     SHCheckMenuItem(theMenu, IDM_VIEW_STATUSBAR, fStatusBarVisible ? TRUE : FALSE);
+
+    /* Disable Auto Arrange when in details or list view */
+    if (SUCCEEDED(fCurrentShellView->QueryInterface(IID_PPV_ARG(IFolderView, &pfv))))
+    {
+        if  (SUCCEEDED(pfv->GetCurrentViewMode(&viewMode)))
```

Comment:
Too much space between "if" and opening parenthesis.

Reply:
Fixed in latest commit